src/components/atoms/Cart.vue

```diff
@@ -0,0 +1,30 @@
+<template>
+	<img v-bind:alt="cart.alt" v-bind:src="cart.src" class="icons__cart">
```

Comment:
why cart compnent have class `icons` at start and not an `cart`, BEM stans for block element modificator, this is cart block, use cart class instead